.github/workflows/static.yml

```diff
@@ -0,0 +1,136 @@
+name: Static
+
+on:
+  push:
+    branches:
+      - 'master'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: true
+
+jobs:
+  run-tests:
+    name: Unit tests and static build
+    uses: ./.github/workflows/run-tests.yml
+  static:
+    name:  Deploy and release Pay tech-doc pages
+    needs: run-tests
```

Comment:
We don't have a run-tests workflow in here 😬 

We should probably add one that just builds the site. I think you can find a run-tests workflow in the pay-product-pages repo too and just copy that into here also